src/lib/routes/client-api/metrics.ts

```diff
@@ -75,6 +78,26 @@ export default class ClientMetricsController extends Controller {
                 }),
             ],
         });
+
+        this.route({
+            method: 'post',
+            path: '/bulk',
+            handler: this.bulkMetrics,
+            permission: NONE, // should have a permission but not bound to any environment
```

Comment:
Should we now consider the permission? Or should we just remove/change the comment

Reply:
Remove comment please. We could have a more descriptive name of the case of client API. In our setup there is a separation of authentication ("do i have access") and authorization ("Can i do this action"). Permission is simply saying there is no specific permissions to perform this action, because the handler needs to implement the filtering itself.

Reply:
Yeah, dangers of copy-pasting. Comment removed.